recipes/chronic/time.patch

```diff
@@ -0,0 +1,27 @@
+From c5f371adcd47a97a63e7edf3a5c92aee17ed5800 Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Wed, 25 Nov 2020 21:29:31 +0100
+Subject: [PATCH] Fix time
```

Comment:
What's the rationale behind this patch?

Reply:
`time.clock` was depreciated in Python 3.3 and has been removed in Python 3.8 https://github.com/python/cpython/pull/13270